examples/timebase_master.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+"""JACK client that set timebase master.
+
+This Client will register itself as `timebase master` and set extended
+position information :
+
+    * BPM
+    * Bar, Beat and Tick according to current position
+    * beat per bar, ticks per beat
+    * beat type, bar start tick
+
+"""
+import jack
+
+BPM = 130
+BAR_START_TICK = 0.0
+BEATS_PER_BAR = 4.0
+BEAT_TYPE = 4.0
+TICKS_PER_BEAT = 960.0
+
+
+def frames2ticks(frame, ticks_per_beat, beats_per_minute, frame_rate):
+    ticks_per_second = (beats_per_minute * ticks_per_beat) / 60
+    return (ticks_per_second * frame) / frame_rate
+
+
+client = jack.Client("Timebase Master")
+
+
+@client.set_timebase_callback
+def timebase_callback(state, nframes, pos, new_pos):
```

Comment:
I think you should check `new_pos`, right?

You should also change `nframes` to `blocksize` (and you will actually need it in the implementation).

See https://github.com/jackaudio/example-clients/blob/master/intime.c.